package.json

```diff
@@ -3,9 +3,10 @@
   "version": "1.0.2",
   "description": "Sophisticated and functionally-minded async with advanced features: coroutines, promises, ES2015 iterables, fantasy-land",
   "main": "dist/creed.js",
-  "jsnext:main": "src/main.js",
+  "jsnext:main": "dist/creed.es.js",
```

Comment:
Yep.  I used the approach [recommended on the rollup wiki](https://github.com/rollup/rollup/wiki/jsnext:main), i.e.:

> you can generate an ES2015 module build alongside your CommonJS/UMD build

I think rollup will still shake the creed tree, since `creed.es.js` is still in es2015 module format (but containing es5 runnable code otherwise).  I was mostly just experimenting, and I'm completely open to other ways to do it.

Sorry, I should have mentioned that I don't really intend to merge this PR as is ... it's kind of a playground right now.